Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -96,12 +96,27 @@ GameList::GameList(QWidget* parent) : QStackedWidget(parent), m_model(this)
   m_prefer_list = Settings::Instance().GetPreferredView();
   ConsiderViewChange();
 
-  auto* zoom_in = new QShortcut(QKeySequence::ZoomIn, this);
-  auto* zoom_out = new QShortcut(QKeySequence::ZoomOut, this);
+  const auto* zoom_in = new QShortcut(QKeySequence::ZoomIn, this);
+  const auto* zoom_out = new QShortcut(QKeySequence::ZoomOut, this);
 
   connect(zoom_in, &QShortcut::activated, this, &GameList::ZoomIn);
   connect(zoom_out, &QShortcut::activated, this, &GameList::ZoomOut);
 
+  // On most keyboards the key to the left of the primary delete key represents 'plus' when shift is
+  // held and 'equal' when it isn't. By common convention, pressing control and that key is treated
+  // treated conceptually as 'control plus' (which is then interpreted as an appropriate zooming
```

Comment:
You copied the word "treated" in here twice.

Reply:
Fixed.